### PR TITLE
fix: multi-auth raise 500 error when use default conf

### PR DESCRIPTION
**Pull request by yuweizzz**
### Description

Fixes: #11156
Related: #9893 

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by shreemaan-abhishek** [comment]
please also add test cases

**Comment by yuweizzz** [comment]
done, pls check again.

**Comment by yuweizzz** [comment]
yes, I cannot reproduce the first question in #9893, and this pr will fix the second question in #9893, I will create a new one.

File: apisix/plugins/multi-auth.lua [hunk 1]
```diff
@@ -54,6 +54,10 @@ function _M.check_schema(conf)
                 if auth.type ~= 'auth' then
                     return false, auth_plugin_name .. " plugin is not supported"
                 end
+                local ok, err = auth.check_schema(auth_plugin_conf, auth.schema)
+                if not ok then
+                    return false, "plugin " .. auth_plugin_name .. " check schema failed: " .. err
```
Comment (shreemaan-abhishek) [inline apisix/plugins/multi-auth.lua:59]: can you also add a test case which shows this error would be raised if any auth_plugin has wrong configuration.

Comment (yuweizzz) [reply apisix/plugins/multi-auth.lua:59]: ok

Comment (shreemaan-abhishek) [reply apisix/plugins/multi-auth.lua:59]: perfect, really appreciate it <3. I will re-review once the CI has completed running.

**Review by moonming — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Pull request merged**